treecreate/webstore
# Feat/#tc-16 initialize admin page

### This pull request closes:

✔️ Closes: #181 
🏁 Jira Issue: [TC-16](https://treecreate.atlassian.net/jira/software/c/projects/TC/boards/1?modal=detail&selectedIssue=TC-16)

### Which service does this issue affect:

- [ ] 🛒 webstore
- [x] 🎓 admin-page

### Types of changes


### Documentation Updates


### Testing checklist

- [x] 💪 Manually tests
- [x] 🔧 Automatic tests

#### Browser compatibility - _**[**Frontend only**]**_


### What changes have been done within this issue?

- Generate a new service called `admin-page` along with a supporting `admin-page-e2e` service
- Adjust the environment and package.json scripts.
- Set the service to run on port `4300`

Used `npx nx generate @nrwl/angular:application --name=admin-page --backendProject=api --routing --strict --no-interactive` to generate the services
### How should this be manually tested?

- Run `nx serve admin-page`, navigate to [localhost:4300](https://localhost:4300) and see if you get a default nx page on 
- Run `npm run e2e:ci`
- Optionally run every script in the package.json

### Screenshots or example usage


`Admin page running on port 4300 showing the default nx page`
![image](https://user-images.githubusercontent.com/22862227/141161568-0ddd0d54-19c6-49ae-9512-a245d8237c3b.png)

---

✅ Closes: #181
